### PR TITLE
Powershell Insights Events cmdlets change

### PR DESCRIPTION
**Pull request by shimedh**
1]. Consolidating the existing cmdlets into a single cmdlet and using parameter set
2]. Support number of events parameter

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@shimedh__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by ogail** [comment]
@shimedh resolve conflicts

**Comment by shimedh** [comment]
Hi Abdelrahman,
                            Should we create a pull request to your master branch or dev branch? What is the difference between the two?

Thanks,
Shibani

**Comment by shimedh** [comment]
Hi Abdelrahman,
                          We have resolved conflicts.

Thanks,
Shibani

**Comment by shimedh** [comment]
+ Andy

From: Shibani Medhekar
Sent: Monday, September 21, 2015 1:54 PM
To: 'Azure/azure-powershell' <reply@reply.github.com>; Azure/azure-powershell <azure-powershell@noreply.github.com>
Subject: RE: [azure-powershell] Powershell Insights Events cmdlets change (#945)

Hi Abdelrahman,
                          We have resolved conflicts.

Thanks,
Shibani

**Comment by stankovski** [comment]
@shimedh resolve conflicts; dev branch is fine

**Comment by stankovski** [comment]
@azuresdkci add to whitelist

**Comment by shimedh** [comment]
Merged Conflicts

**Comment by markcowl** [comment]
@azuresdkci retest this please

**Comment by stankovski** [comment]
@azuresdkci retest this please

File: src/ResourceManager/Insights/Commands.Insights/Events/GetAzureLogCommand.cs [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Insights.OutputClasses;
+
+namespace Microsoft.Azure.Commands.Insights.Events
+{
+    /// <summary>
+    /// Get the list of events for at a subscription level.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "AzureLog"), OutputType(typeof(List<IPSEventData>))]
+    public class GetAzureLogCommand : EventCmdletBase
```
Comment (stankovski) [inline src/ResourceManager/Insights/Commands.Insights/Events/GetAzureLogCommand.cs]: All ResourceManager cmdlets should have AzureRm in the name - e.g. Get-AzureRmLog

**Pull request merged**